site-service/src/test/java/uk/ac/ox/ndph/mts/site_service/ApplicationTest.java

```diff
@@ -0,0 +1,23 @@
+package uk.ac.ox.ndph.mts.site_service;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.ac.ox.ndph.mts.site_service.Application;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+
+@ExtendWith(MockitoExtension.class)
+class ApplicationTest {
+    final String strStartupName = "STARTUP";
+    final String strStartupValue = "Staring site service...";
+
+    @Test
+    void messageAssert() {
+        Application app = Application.STARTUP;
+        assertThat(app.name(), equalTo(strStartupName));
+        assertThat(app.message(), equalTo(strStartupValue));
+    }
```

Comment:
Not sure I agree with the existence of this test. It's saying that the enum value is *required* to be STARTUP, and the associated message is *required* to be "Staring site service...".  I don't think these are requirements, I think they're implementation details. Unit tests declare a certainty that something *should* be the case.

If someone refactors the enum value or message to be something else then the test will break (even though the code will still work), and they will then be in a problematic situation, asking "why does the test not want me to touch these things? Am I missing something?"

Reply:
Removed ApplicationTest class